diff -ur BoulderDash-1.0-b5-a/configure BoulderDash-1.0-b5-b/configure
--- BoulderDash-1.0-b5-a/configure	1998-09-08 22:08:12.000000000 +0200
+++ BoulderDash-1.0-b5-b/configure	2012-06-08 20:58:17.580519262 +0200
@@ -2826,7 +2826,7 @@
    *)         ;;
 esac
 
-if test "$CC" = "gcc"; then
+if [[ "$CC" = *gcc* ]]; then
    CFLAGS="$CFLAGS -Wall -Wshadow -Wstrict-prototypes -Wmissing-prototypes"
    CFLAGS="$CFLAGS -Wmissing-declarations -Wformat -Wchar-subscripts -fno-strength-reduce"
 
diff -ur BoulderDash-1.0-b5-a/src/commodore.h BoulderDash-1.0-b5-b/src/commodore.h
--- BoulderDash-1.0-b5-a/src/commodore.h	1998-09-06 05:13:00.000000000 +0200
+++ BoulderDash-1.0-b5-b/src/commodore.h	2012-06-08 20:59:20.359730020 +0200
@@ -1,6 +1,7 @@
 #ifndef _COMMODORE_H_
 #define _COMMODORE_H_
 
+#include "objects.h"
 
 extern struct OBJECT_DATA commodore_objects_data[];
 extern unchar             commodore_cave_raw_data[80][CAVE_RAW_MAXSIZE];
