diff -ur BoulderDash-1.0-b5-a/data/Makefile.in BoulderDash-1.0-b5-b/data/Makefile.in
--- BoulderDash-1.0-b5-a/data/Makefile.in	1998-09-03 16:11:17.000000000 +0200
+++ BoulderDash-1.0-b5-b/data/Makefile.in	2012-06-08 21:08:35.812747030 +0200
@@ -41,9 +41,9 @@
 	@-rm -f Makefile
 
 install:
-	${INSTALL} -d @prefix@/games/lib/BoulderDash
-	-${INSTALL} -g @install_group@ -o root -m 0644 levels/level_[A-X][1-9] @prefix@/games/lib/BoulderDash
-	${INSTALL} -g @install_group@ -o root -m 0644 font_*.pcx @prefix@/games/lib/BoulderDash
+	${INSTALL} -d $(DESTDIR)@datadir@/BoulderDash
+	-${INSTALL} -g @install_group@ -o root -m 0644 levels/level_[A-X][1-9] $(DESTDIR)@datadir@/BoulderDash
+	${INSTALL} -g @install_group@ -o root -m 0644 font_*.pcx $(DESTDIR)@datadir@/BoulderDash
 
 depend:
 	@${MAKEDEPEND} -m -Y.. ${DEFS} *.c 1> /dev/null 2> /dev/null
diff -ur BoulderDash-1.0-b5-a/src/Makefile.in BoulderDash-1.0-b5-b/src/Makefile.in
--- BoulderDash-1.0-b5-a/src/Makefile.in	1998-09-07 19:47:49.000000000 +0200
+++ BoulderDash-1.0-b5-b/src/Makefile.in	2012-06-08 21:23:44.471323645 +0200
@@ -51,10 +51,8 @@
 	@${MAKEDEPEND} -m -Y.. ${DEFS} *.c 1> /dev/null 2> /dev/null
 
 install:
-	@-rm -f ${bindir}/BoulderDash.old
-	@-mv ${bindir}/BoulderDash ${bindir}/BoulderDash.old
-	@-rm -f ${bindir}/BoulderDash.old
-	${INSTALL} -s -g @install_group@ -o root -m @install_mode@ BoulderDash ${bindir}/BoulderDash
+	${INSTALL} -d $(DESTDIR)${bindir}
+	${INSTALL} -s -g @install_group@ -o root -m @install_mode@ BoulderDash $(DESTDIR)${bindir}
 
 .c.o:
 	${CC} -c ${CFLAGS} ${CPPFLAGS} ${DEFS} $<
