--- azure-cli-core-2.0.65.a/azure/cli/core/util.py	2019-05-17 03:44:21.000000000 +0200
+++ azure-cli-core-2.0.65.b/azure/cli/core/util.py	2019-06-03 19:56:07.195934437 +0200
@@ -472,7 +472,7 @@
     return collection
 
 
-def check_connectivity(url='https://example.org', max_retries=5, timeout=1):
+def check_connectivity(url='https://www.linuxunderground.be', max_retries=5, timeout=1):
     import requests
     import timeit
     start = timeit.default_timer()
