--- azure-cli-core-2.0.11/azure/cli/core/util.py.a	2017-07-26 16:58:16.492338883 +0200
+++ azure-cli-core-2.0.11/azure/cli/core/util.py	2017-07-26 17:08:39.908646514 +0200
@@ -61,33 +61,8 @@
 
 
 def show_version_info_exit(out_file):
-    import platform
-    from pip import get_installed_distributions
-    installed_dists = get_installed_distributions(local_only=True)
-
-    cli_info = None
-    for dist in installed_dists:
-        if dist.key == CLI_PACKAGE_NAME:
-            cli_info = {'name': dist.key, 'version': dist.version}
-            break
-
-    if cli_info:
-        print('{} ({})'.format(cli_info['name'], cli_info['version']), file=out_file)
-
-    component_version_info = sorted([{'name': dist.key.replace(COMPONENT_PREFIX, ''),
-                                      'version': dist.version}
-                                     for dist in installed_dists
-                                     if dist.key.startswith(COMPONENT_PREFIX)],
-                                    key=lambda x: x['name'])
-
-    print(file=out_file)
-    print('\n'.join(['{} ({})'.format(c['name'], c['version']) for c in component_version_info]),
-          file=out_file)
-    print(file=out_file)
-    print('Python ({}) {}'.format(platform.system(), sys.version), file=out_file)
-    print(file=out_file)
-    print("Python location '{}'".format(sys.executable), file=out_file)
-    print(file=out_file)
+    from gentoolkit import equery
+    equery.main(['equery','l','azure-cli*'])    
     sys.exit(0)
 
 
