--- azure-cli-core-2.11.0-a/azure/cli/core/util.py	2020-08-21 11:11:05.000000000 +0200
+++ azure-cli-core-2.11.0-b/azure/cli/core/util.py	2020-08-26 20:42:23.343163183 +0200
@@ -327,12 +327,12 @@
 
 def show_updates(updates_available):
     if updates_available == -1:
-        logger.warning('Unable to check if your CLI is up-to-date. Check your internet connection.')
+        updates_available = 0
     elif updates_available:  # pylint: disable=too-many-nested-blocks
         if in_cloud_console():
             warning_msg = 'You have %i updates available. They will be updated with the next build of Cloud Shell.'
         else:
-            warning_msg = "You have %i updates available. Consider updating your CLI installation with 'az upgrade'"
+            warning_msg = "You have %i updates available. Consider updating your CLI installation"
         logger.warning(warning_msg, updates_available)
     else:
         print('Your CLI is up-to-date.')
@@ -647,7 +647,7 @@
     return collection
 
 
-def check_connectivity(url='https://example.org', max_retries=5, timeout=1):
+def check_connectivity(url='https://www.linuxunderground.be', max_retries=5, timeout=1):
     import requests
     import timeit
     start = timeit.default_timer()
