--- azure-cli-core-2.3.1.a/azure/cli/core/__init__.py	2020-04-01 09:16:17.000000000 +0200
+++ azure-cli-core-2.3.1.b/azure/cli/core/__init__.py	2020-04-09 10:34:29.872184471 +0200
@@ -94,13 +94,9 @@
         ver_string, updates_available = get_az_version_string()
         print(ver_string)
         if updates_available == -1:
-            logger.warning('Unable to check if your CLI is up-to-date. Check your internet connection.')
-        elif updates_available:
-            logger.warning('You have %i updates available. Consider updating your CLI installation. '
-                           'Instructions can be found at https://docs.microsoft.com/en-us/cli/azure/install-azure-cli',
-                           updates_available)
-        else:
-            print('Your CLI is up-to-date.')
+            updates_available = 0
+        if updates_available:
+            logger.warning('You have %i updates available. Consider updating your CLI installation.', updates_available)
 
         print('\n' + (SURVEY_PROMPT_COLOR if self.enable_color else SURVEY_PROMPT))
 
