--- azure-cli-core-2.0.78-a/azure/cli/core/__init__.py	2019-12-12 12:06:28.000000000 +0100
+++ azure-cli-core-2.0.78-b/azure/cli/core/__init__.py	2019-12-21 20:42:41.920617565 +0100
@@ -89,13 +89,9 @@
         ver_string, updates_available = get_az_version_string()
         print(ver_string)
         if updates_available == -1:
-            logger.warning('Unable to check if your CLI is up-to-date. Check your internet connection.')
-        elif updates_available:
-            logger.warning('You have %i updates available. Consider updating your CLI installation. '
-                           'Instructions can be found at https://docs.microsoft.com/en-us/cli/azure/install-azure-cli',
-                           updates_available)
-        else:
-            print('Your CLI is up-to-date.')
+            updates_available = 0
+        if updates_available:
+            logger.warning('You have %i updates available. Consider updating your CLI installation.', updates_available)
 
         print('\n' + SURVEY_PROMPT)
 
