--- ti99sim-0.14.0-a/rules.mak	2016-06-26 02:33:32.000000000 +0200
+++ ti99sim-0.14.0-b/rules.mak	2026-05-17 15:17:32.032448407 +0200
@@ -37,17 +37,12 @@
 DEBUG    := 1
 endif
 
-ifdef ARCH
-CFLAGS   += -march=$(ARCH)
-endif
-
 ifdef DEBUG
 CFLAGS   += -g3 -O0 -DDEBUG
-else
-CFLAGS   += -g3 -O3 -fomit-frame-pointer
-LFLAGS   += -g3 -O3
 endif
 
+CFLAGS   += -DTI_DATA_DIR=\"$(DATA_DIR)\"
+
 ifdef LTO
 CFLAGS   += -flto
 LFLAGS   += -flto
