--- ti99sim-0.15.0-a/rules.mak	2017-03-18 23:19:20.000000000 +0100
+++ ti99sim-0.15.0-b/rules.mak	2026-05-17 15:14:38.519765081 +0200
@@ -16,7 +16,7 @@
 CXX      := ccache $(CXX)
 endif
 
-CFLAGS   += --std=c++11
+CFLAGS   += -std=c++20
 CFLAGS   += -fno-strict-aliasing -fexceptions -fPIC
 CFLAGS   += -fdata-sections -ffunction-sections
 CFLAGS   += -funsigned-char
@@ -37,10 +37,6 @@
 DEBUG    := 1
 endif
 
-ifdef ARCH
-CFLAGS   += -march=$(ARCH)
-endif
-
 ifdef LTO
 CFLAGS   += -flto
 LFLAGS   += -flto
@@ -48,11 +44,10 @@
 
 ifdef DEBUG
 CFLAGS   += -g3 -O0 -DDEBUG
-else
-CFLAGS   += -g3 -O3 -fomit-frame-pointer
-LFLAGS   += -g3 -O3
 endif
 
+CFLAGS   += -DTI_DATA_DIR=\"$(DATA_DIR)\"
+
 ifdef LOGGER
 CFLAGS   += -D_REENTRANT -DLOGGER
 LIBS     += -lrt
