--- configure_ori.ac	2013-09-18 14:00:45.000000000 +0200
+++ configure.ac	2013-09-19 16:59:25.000000000 +0200
@@ -80,7 +80,7 @@
 PKG_CHECK_MODULES([libpng],     [libpng])
 PKG_CHECK_MODULES([sndfile],    [sndfile])
 PKG_CHECK_MODULES([lua], [lua], [], [ PKG_CHECK_MODULES([lua], [lua5.1]) ] )
-PKG_CHECK_MODULES([json], [json], [], [AC_MSG_ERROR(['json-c' not found error])])
+PKG_CHECK_MODULES([json], [json-c], [], [AC_MSG_ERROR(['json-c' not found error])])
 
 AM_PATH_LIBMIKMOD([], [], [
   AC_MSG_ERROR([unable to find libmikmod])
