--- configure_ori.ac	2013-10-03 20:19:26.036416606 +0200
+++ configure.ac	2013-10-03 20:20:24.635679914 +0200
@@ -43,10 +43,6 @@
 AC_PROG_GREP
 AC_PROG_EGREP
 AC_PROG_SED
-AC_CHECK_PROG([convert], [convert], [convert])
-if test "x$ac_cv_prog_convert" == "x" ; then
-  AC_MSG_ERROR(['convert' not found error, install ImageMagick.])
-fi
 AC_CHECK_PROG([xcf2png], [xcf2png], [xcf2png])
 if test "x$ac_cv_prog_xcf2png" == "x" ; then
   AC_MSG_ERROR(['xcf2png' not found error, install XCFTools.])
